third_party/move/tools/move-unit-test/tests/move_unit_test_testsuite.rs

```
@@ -55,6 +55,7 @@ fn run_test_with_modifiers(
                     None,
                     buffer,
                     UnitTestFactoryWithCostTable::new(None, None),
+                    unit_test_config.fail_fast,
```

Can you please add some tests to showcase fail fast? The tests can be added in this crate. Some cases to consider when adding tests:
1. multiple tests fail, but with fail fast, we stop after the first error
2. multiple tests fail, but without fail fast, all failures are reported
3. some tests are expected to fail with a specific abort. with fail fast, we don't stop (and all tests "pass" with the appropriate failure), because these are all expected errors.

---

done

---

It seems `3` is not added yet. Am I misunderstanding it?

---

there is a test that is expected to fail, and is correctly handled. See

```
    #[test, expected_failure(abort_code = TWO)]
    fun test_foo() {
        let ret = bar(19);
        assert!(ret == 17, TWO);
    }
```